tools/library/include/cutlass/library/descriptions.h

```diff
@@ -34,7 +34,7 @@
 #include <cutlass/library/types.h>
 #include <cutlass/blas3_types.h>
 #include <cutlass/gemm_coord.h>
-
+#include <stdexcept>
```

Comment:
Include for `std::runtime_error` used in this file. This will allow us to include this file in other projects without have `#include <stdexcept>` above its include.